src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1375,38 +1371,6 @@ protected bool CheckAllParamsAreTypeOrSingleColumnTable(CheckTypesContext contex
 
         #endregion
 
-        internal TransportSchemas.FunctionInfo Info(string locale)
```

Comment:
>Info [](http://example.com/codeflow?start=47&length=4)

what was this used for before?

Reply:
Not by us.
Maybe PA Client and I think this code could be moved there (if used, or course)